### PR TITLE
(WIP)stage: show stages

### LINKED ISSUE #4488: stage: show stages
## Summary

```
show stages
```

Show all the stage names of a tenant.

### PR DESCRIPTION
**Pull request by LiuYuHui**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

## Changelog

- New Feature

## Related Issues

Fixes #4488

### CODE REVIEW

File: common/protos/proto/user.proto [hunk 1]
```diff
@@ -178,4 +178,6 @@ message UserStageInfo {
   CopyOptions copy_options = 5;
 
   string comment = 6;
+
+  string created_on = 7;
```
Comment (BohuTANG) [inline common/protos/proto/user.proto:182]: To keep backwards compatible,it seems  need 'Option' here

Comment (Xuanwo) [reply common/protos/proto/user.proto:182]: I don't like the idea to store a time string in metasrv.

How about converting to [Timestamp](https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/timestamp.proto) or UNIX timestamp as u64.

Comment (LiuYuHui) [reply common/protos/proto/user.proto:182]: Good suggestion, will try it :D

**Comment by LiuYuHui** [comment]
close this PR, as #5581 has better implementation than me :(.

**Pull request closed without merging**